### ISSUE TITLE
Feature request: backup/restore action

### ISSUE THREAD

**Issue by dmzoneill** [Juniper/contrail-charms#109]
Ideally the charm should provide a juju action for backups [1].
this would allow administrators to run a backup/restore operation without underlying knowledge.

Operation would looks something like

juju run-action contrail-<service>/leader backup --to file
juju run-action contrail-<service>/leader restore --form file

[1] https://jaas.ai/docs/working-with-actions

**Comment by Andrey-mp** [comment]
Yeah, it's a good enhancement. It can be a jira-ticket in internal juniper's jira or in TF's jira.

**Comment by dmzoneill** [comment]
Sure Andrey

If you can point me at the Jira I can create the story.

thanks

**Comment by Andrey-mp** [comment]
Hi Dave,
I don't know about Juniper's Jira - I suppose it can be accessed only by Juniper's employee.
TF's jira is here - https://jira.tungsten.io/secure/Dashboard.jspa
